packages/jellyfish-api-core/__tests__/category/spv/createAnchor.test.ts

```diff
@@ -0,0 +1,242 @@
+      amount: 15800,
+      privkey: 'b0528d87cfdb09f72c9d10b7b3cc00727062d93537a3e8abcf1fde821d08b59d'
+    }], 'INVALID')
+
+    await expect(promise).rejects.toThrow(RpcApiError)
+    await expect(promise).rejects.toThrow('rewardAddress (INVALID) does not refer to a P2PKH or P2WPKH address')
+  })
+
+  it('should createAnchor', async () => {
+    const rewardAddress = await clients[0].spv.getNewAddress()
+
+    const anchor = await clients[0].spv.createAnchor([{
+      txid: '11a276bb25585f6973a4dd68373cffff41dbcaddf12bbc1c2b489d1dc84564ee',
+      vout: 2,
+      amount: 15800,
+      privkey: 'b0528d87cfdb09f72c9d10b7b3cc00727062d93537a3e8abcf1fde821d08b59d'
+    }], rewardAddress)
+    expect(typeof anchor.txHex).toStrictEqual('string')
+    expect(typeof anchor.txHash).toStrictEqual('string')
+    expect(typeof anchor.defiHash).toStrictEqual('string')
+    expect(anchor.defiHeight).toStrictEqual(15)
+    expect(anchor.estimatedReward).toStrictEqual(new BigNumber(0))
+    expect(anchor.cost).toStrictEqual(new BigNumber(3556))
+    expect(anchor.sendResult).toStrictEqual(0)
+    expect(anchor.sendMessage).toStrictEqual('Success')
+
+    {
+      const pending = await group.get(0).call('spv_listanchorspending')
+      // pending does not contain active property
+      expect(pending.active).toBeUndefined()
+    }
+
+    // generate the anchor block
+    await group.get(0).generate(1)
+
+    // pending is cleared
+    {
+      const pending = await group.get(0).call('spv_listanchorspending')
+      expect(pending.length).toStrictEqual(0)
+    }
+
+    // should be not active yet
+    const anchors = await group.get(0).call('spv_listanchors')
+    expect(anchors.length).toStrictEqual(1)
+    expect(anchors[0].btcBlockHeight).toStrictEqual(0)
+    expect(typeof anchors[0].btcBlockHash).toStrictEqual('string')
+    expect(typeof anchors[0].btcTxHash).toStrictEqual('string')
+    expect(typeof anchors[0].previousAnchor).toStrictEqual('string')
+    expect(anchors[0].defiBlockHeight).toStrictEqual(15)
+    expect(typeof anchors[0].defiBlockHash).toStrictEqual('string')
+    expect(anchors[0].rewardAddress).toStrictEqual(rewardAddress)
+    expect(anchors[0].confirmations).toBeLessThan(6)
+    expect(anchors[0].signatures).toStrictEqual(2)
+    expect(anchors[0].anchorCreationHeight).toStrictEqual(60)
+    expect(anchors[0].active).toStrictEqual(false)
+
+    // to activate anchor at min 6 conf
+    await group.get(0).call('spv_setlastheight', [6])
+
+    {
+      // should be active now
+      const anchors = await group.get(0).call('spv_listanchors')
+      expect(anchors[0].confirmations).toBeGreaterThanOrEqual(6)
+      expect(anchors[0].active).toStrictEqual(true)
+    }
+
+    {
+      // auths back to zero after the anchor active
+      const auths = await group.get(0).call('spv_listanchorauths')
+      expect(auths.length).toStrictEqual(0)
+    }
+  })
+
+  it('should be failed as min anchor quorum was not reached', async () => {
```

Comment:
This test depends on `should createAnchor`. tests should be independent. maybe update the test so that first create an anchor. then within the same test try to create another anchor by the same node.